test/config.test.js

```diff
@@ -72,6 +72,50 @@ describe('environment config', () => {
 
 });
 
+describe('`fozzieSettings` config', () => {
```

Comment:
Could you keep these tests in the same format as the existing tests, please? Makes it much easier to scan through and understand the tests.

Reply:
Do you mean adding the comments or removing the describe blocks?

Reply:
The other setting use the same description wording ("zzz should be set" & "zzz can be updated") and use the AAA comment separation.

Reply:
Hmm. I'm not sure following the exact same description here/all the time makes sense? I have a property that shouldn't be changed within this config file so wanted to make that clear in the tests so if it's changed in the config they're fed back with a reason why they shouldn't update it here and not just a failing test. Although the comments make it more readable I think we should avoid them IMHO because they should be self explanatory/readable without them, i.e `Assertion is what expect does` I would have thought? (Will add them anyway). The reason I didn't follow the AAA and the description is because `pathbuilder.tests.js` uses a different setup for AAA/describe. Might be a good idea to discuss the way we write tests in the future because I find it harder to understand the current format. For example: I've found it nicer and easier to understand when things are nested to individual properties or methods that belong to the top most scope within describe blocks, because if you have a failing test you can locate it easily (also easier to find where to added further tests), right now if a test fails you have to scan the entire file to find the top level describe (currently not scoped to individual properties) and then scan through the multiple `IT` statement which is causing the failure. It's also easier to see which scope is broken on the command line.

Reply:
The main thing I'm concerned with when it comes to the tests is keeping them consistent, easy for newcomers to pick up, and steering clear of complex logic. 

Avoiding DRY, sticking to the AAA approach, and asserting against the DOM pretty much solves those issues. As for the `describe` block`, I'm not against using them, I just find that nesting them too much makes the test file harder to follow, and in a recent instance in Global Web, the `describe` blocks made it made it really difficult to find the failing tests. I guess it's a case of using them sensibly when it makes sense?

Happy to discuss this as a group, maybe at the next UI Guild meeting?

Reply:
@kevinrodrigues As someone new to structuring tests (noob alert 🚨 ) best thing to do if you think there is a clearer way to structure the tests would be to make a small (separate) PR example of how you would structure them so we can compare between them (maybe discuss at a UI guild or in a separate meeting once this is done).

As someone with no real affiliation to any structure, happy to then give my opinion 😄

Reply:
@ashleynolan @DamianMullins  I'll put one example together so we can discuss it in one of the UI meetings 👍